pkg/binary/envoy/debug/node.go

```diff
@@ -141,3 +143,29 @@ func networkInterfaces(r binary.Runner) error {
 
 	return nil
 }
+
+// writeIOStat write iostat of devices in the form of a dictionary to json file
+func writeIOStats(r binary.Runner) error {
+	f, err := os.Create(filepath.Join(r.DebugStore(), "node/iostats.json"))
+	defer f.Close() //nolint
+	if err != nil {
+		return fmt.Errorf("error in creating iostat.json: %v", err)
+	}
+
+	physicalPartitions, _ := d.Partitions(false)
+	deviceNames := make([]string, 0, len(physicalPartitions))
+	//nolint:gocritic
```

Comment:
What was the complaint here? CopyRange?

Reply:
Yes, the error `rangeValCopy: each iteration copies 64 bytes (consider pointers or indexing) (gocritic)` is returned

Reply:
nit: Should we ignore the lint error? I'm not familiar with Golang, I think we should follow linters in most of the cases, here we can avoid the copy like:

```
for i := range physicalPartitions {
  deviceNames = append(deviceNames, physicalPartitions[i].Device)
}
```

Reply:
Thank you. That's a really good suggestion.